frontend/src/components/welcome/components/Info.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+import Chip from 'material-ui/Chip'
+import Avatar from 'material-ui/Avatar'
+import Typography from 'material-ui/Typography'
+
+const data = {
+    tasks: {text: 'Total of Tasks', value: '--'},
+    orders: {text: 'Bounties Paid', value: '--'},
+    users: {text: 'Total of Users', value: '--'}    
+}
+
+const Content = styled.span`
+   padding-bottom: 10px;
+   color: white;
+   text-align: center;
+`
+
+const Items = styled.div`
+  margin-top: 20px;
+`
+
+const Item = styled(Chip)`
+display:
+  outline: 1px solid orange;
+  margin: 10px;
+  font-weight: bold;
+  width: 140px;
+  justify-Content: space-between !important;
+`
+
+const Icon = styled(Avatar)`
+width: 90px !important;
+font-size: 0.8rem !important;
+border-radius: 16px !important;
+`
+class Info extends React.Component {
+  constructor (props) {
+    super(props)
+    
```

Comment:
You're not using this state anywhere in the component, so I think maybe this is not necessary here and you have now a connected component, so it will re-render with the updated data as the redux changes.